public/js/components/RightSidebar.js

```diff
@@ -192,13 +192,13 @@ const RightSidebar = React.createClass({
 
   getItems() {
     const items = [
-    { header: L10N.getStr("breakpoints.header"),
-      component: Breakpoints,
-      opened: true },
-    { header: L10N.getStr("callStack.header"),
-      component: Frames },
-    { header: L10N.getStr("scopes.header"),
-      component: Scopes }
+      { header: L10N.getStr("breakpoints.header"),
+        component: Breakpoints,
+        opened: true },
+      { header: L10N.getStr("callStack.header"),
+        component: Frames },
+      { header: L10N.getStr("scopes.header"),
+        component: Scopes }
```

Comment:
I am subjectively on the side of expanding the braces, I feel guilt that it takes up so much room but find it much easier to read. 😄